src/client/driver/command-executors/execute-action-command.js

```diff
@@ -148,6 +152,15 @@ function ensureCommandElements (command, timeout) {
         });
 }
 
+function ensureCommandArguments (command) {
+    if (command.type === COMMAND_TYPE.pressKey) {
+        var parsedKeySequence = parseKeySequence(command.keys);
```

Comment:
The only browser-specific part of `KEY_MAPS` is `symbolCharCodeToKeyCode`. Just the one condition for `Firefox`, that's silly. Maybe there is some other way to do it.